setup.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
It does not detect all the possible paths to recommend installing the executable there, but we can start with this as a baseline